build.gradle

```diff
@@ -246,7 +253,9 @@ dependencies {
     testImplementation "org.testfx:testfx-junit5:4.0.16-alpha"
     testImplementation "org.hamcrest:hamcrest-library:2.2"
 
-    checkstyle 'com.puppycrawl.tools:checkstyle:10.12.3'
+    checkstyle('com.puppycrawl.tools:checkstyle:10.12.3') {
+        exclude group: 'org.antlr'
```

Comment:
this is wrong, we cannot remove it here and checkstyle is never added to the compile path /runtime path

Reply:
Fixed. -- Wrong use of IntelliJ dependency analysis...